modules/serverless_negs/variables.tf

```diff
@@ -85,6 +85,15 @@ variable "backends" {
       description = optional(string)
 
     }))
+
+    // serverless_neg_backends is mutually exclusive to groups.There can only be one serverless neg per region
+    // with one of cloud-run, cloud-functions and app-engine as service.
+    serverless_neg_backends = list(object({
+      region  = string,
+      type    = string, // cloud-run, cloud-function and app-engine
```

Comment:
Could we add a variable validation rule to ensure expected type

Reply:
Added validation!